src/cljs/rems/actions/add_member.cljs

```diff
@@ -26,9 +26,17 @@
 (rf/reg-event-db
  ::set-potential-members
  (fn [db [_ members]]
-   (assoc db
-          ::potential-members (set (map atoms/enrich-user members))
-          ::selected-member nil)))
+   (let [application (-> db :rems.application/application :data)
+         applicant (-> application :application/applicant)
+         existing-ids (set (concat 
+                            [(:userid applicant)] 
+                            (map :userid (:application/members application))))
+         filtered (->> (set members)
+                       (filter #(not (contains? existing-ids (:userid %))))
```

Comment:
Something to consider here:

`(filter #(not (contains? existing-ids (:userid %))))` 

could be written simpler as 

`(remove #(contains? existing-ids (:userid %)))`

or even like

`(remove (comp existing-ids :userid))`

The use of `remove` removed the need of negation. And using `(comp set ...)` is an example of the search idiom in Clojure which is also quite common in REMS codebase.